samples/open-telemetry/jaeger/Core_8/Publisher/Publisher.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net6.0</TargetFramework>
+		<ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Probably, like me, need to add  `<LangVersion>7.3</LangVersion>` which means removing implicit using and some other small refactorings.